parlai/agents/example_seq2seq/example_seq2seq.py

```diff
@@ -150,107 +145,82 @@ def share(self):
 
         return shared
 
-    def predict(self, xs, ys=None, is_training=False):
-        """Produce a prediction from our model.
-
-        Update the model using the targets if available.
-        """
+    def v2t(self, vector):
+        """Converts vector to text."""
+        if vector.dim() == 1:
+            output_tokens = []
+            # Remove the final END_TOKEN that is appended to predictions
+            for i, token in enumerate(vector):
+                if token == self.END_IDX:
```

Comment:
Do we also need to check for START_IDX?

Reply:
eh this is simpler and it shouldn't actually output START_IDX if it's learning properly

Reply:
(y)